SimGeneral/MixingModule/plugins/MixingModule.cc

```diff
@@ -219,6 +219,7 @@ namespace edm {
     std::vector<std::string> digiNames = digiPSet.getParameterNames();
     for(auto const& digiName : digiNames) {
         ParameterSet const& pset = digiPSet.getParameterSet(digiName);
+	consumes<HepMCProduct>(pset.getParameter<edm::InputTag>("HepMCProductLabel"));
```

Comment:
@jingyucms - i'm not sure why this line is here. If its needed, it should be outside the loop